pkgs/top-level/all-packages.nix

```diff
@@ -2332,6 +2332,10 @@ with pkgs;
 
   flashrom = callPackage ../tools/misc/flashrom { };
 
+  flent = callPackage ../applications/networking/flent {
```

Comment:
you can use python3Packages.callPackage instead and it will pass the correct parameters without you expliciting matplolib & co.